src/libs/actions/BankAccounts.js

```diff
@@ -57,7 +57,7 @@ function clearOnfidoToken() {
 
 /**
  * Helper method to build the Onyx data required during setup of a Verified Business Bank Account
- * @param {String | undefined} currentStep
+ * @param {String | undefined} currentStep The step that we need to update the data from BE to draft value
```

Comment:
```suggestion
 * @param {String | undefined} currentStep The step that we need to update the data from backend to draft value
```

Is this what it means? If yes, let's update it to use the full word, I don't find it necessary to save a few characters. And if yes, I honestly don't really understand the sentence, perhaps you can try rephrasing it? 

Also, you did not clarify what happens if you pass `undefined`?

Reply:
>I think something like The name of the step for which we will update the draft value when data has been sucessfully submitted to the backend. would be clearer.

Agreed, I do understand this better. Also, can we say "API" instead of "backend", since that's what we use everywhere?